### ISSUE TITLE
GridFieldEditableColumns 

### ISSUE THREAD

**Issue by sachithra-gayan** [symbiote/silverstripe-gridfieldextensions#189]
hello,

I got an error when using GridFieldEditableColumns  in ss4.

```
[Recoverable Error] Argument 1 passed to SilverStripe\Forms\Form::__construct() must be an instance of SilverStripe\Control\RequestHandler, instance of SilverStripe\GridFieldExtensions\GridFieldEditableColumns given, called in /var/www/html/straynew/gridfieldextensions/src/GridFieldEditableColumns.php on line 278 and defined
```

**Comment by sachithra-gayan** [comment]
https://github.com/silverstripe-australia/silverstripe-gridfieldextensions/pull/190

**Comment by robbieaverill** [comment]
Fixed in #190
